### PR TITLE
Fixed #684 - We are Hiring banner reappearing issue

### LINKED ISSUE #684: "We are Hiring" Banner reappearing in every pages after closing the banner
Description:
"We are Hiring" Banner reappearing in every pages after closing the banner
Expectation:
"We are Hiring" Banner should not reappear n every pages after closing

### PR DESCRIPTION
**Pull request by vaibhav111tandon**
#### Summary

Fixed #684 Reappearing "We are Hiring" banner issue on every page reload.

#### Ticket Link

  Fixes #684

### CODE REVIEW

File: site/static/js/main.js [hunk 1]
```diff
@@ -8,8 +8,17 @@ $(document).ready(function(){
...
+    }
+
     $('.notification-bar__close').on('click', function(){
-       $(".notification-bar").addClass("closed");
-       $("header").removeClass("with-notification-bar");
+        if(!window.name){
+            $(".notification-bar").addClass("closed");
+            $("header").removeClass("with-notification-bar");
+        }
+        window.name = true;
```
Comment (hanzei) [inline site/static/js/main.js:21]: It seems odd to use `window.name` for this purpose. Could you please elaborate why you choose this approach vs using a Cookie?

**Review by hanzei — CHANGES_REQUESTED** [review]

File: site/static/js/main.js (hunk 1 above)
Comment (vaibhav111tandon) [reply site/static/js/main.js:21]: No specific reason as such...we can go with session cookie as well

**Comment by vaibhav111tandon** [comment]
Done changes

File: site/static/js/main.js [hunk 2]
```diff
@@ -8,8 +8,18 @@ $(document).ready(function(){
...
+    }
+
     $('.notification-bar__close').on('click', function(){
-       $(".notification-bar").addClass("closed");
-       $("header").removeClass("with-notification-bar");
+        if(!notifFlagExist){
+            $(".notification-bar").addClass("closed");
+            $("header").removeClass("with-notification-bar");
+        }
+        document.cookie = 'notifFlag=true'
```
Comment (hanzei) [inline site/static/js/main.js:23]: Nit pick on namin: Should this be `hideBanner` instead?

**Review by hanzei — APPROVED** [review]
LGTM. One question about naming

File: site/static/js/main.js (hunk 2 above)
Comment (vaibhav111tandon) [reply site/static/js/main.js]: Done

**Review by asaadmahmood — APPROVED** [review]

**Pull request merged**